### DISCUSSION TITLE
使用resolutions后出现打包过慢、包体积增大。

### DISCUSSION THREAD

**Discussion by BeautyInNorth** [umijs/umi#8711] [Q&A]
```
"resolutions": {
  "**/af-webpack/typescript": "3.7.2"
}
```
在不升级Umi版本的情况下，使用可选链操作符。在通过resolutions覆盖TS低版本后，出现打包编译变慢，出包体积增大的问题。

**Comment by fz6m** [comment]
由于缺乏足够的信息，请提供 [最小重现](https://stackoverflow.com/help/minimal-reproducible-example) 的 demo 帮助排查问题。
